src/web/components/ApiKeyManagement/KeyCreationDialog.tsx

```diff
@@ -39,7 +39,7 @@ function CreateApiKeyForm({ onFormSubmit, availableRoles, closeDialog }: CreateA
   const { handleSubmit } = formMethods;
   return (
     <>
-      <h1>Add API Key</h1>
+      <h1 className='dialog-title'>Add API Key</h1>
```

Comment:
The reason that the Dialog cannot have the title passed to it like dialogs in other files is because the dialog alternates depending on if we are showing the secret or not, and we don't want the title to be the same for both instances